flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/logical/operators.scala

```diff
@@ -652,6 +653,21 @@ case class WindowAggregate(
       case ValidationSuccess => // ok
     }
 
+    // validate property
+    if (propertyExpressions.nonEmpty) {
+      resolvedWindowAggregate.window match {
+        case TumblingGroupWindow(_, _, size) if isRowCountLiteral(size) =>
+          failValidation("Window start and Window end are not accessed " +
+                           "on row-counts Tumbling window.")
```

Comment:
How about `"Window start and Window end cannot be selected for a row-count Tumbing window."`?